app/controllers/item.js

```diff
@@ -38,9 +37,10 @@ export default Ember.Controller.extend({
   }),
 
   hasDraftOrder: Ember.computed.alias("session.draftOrder"),
-  isOrderFulfilmentUser: Ember.computed(function() {
+  isOrderFulfilmentUser: Ember.computed('session.authToken', function() {
     let user = this.get('session.currentUser');
-    return user.hasRole('Order fulfilment');
+    let isLoggedIn = this.get('session.isLoggedIn');
+    return user && isLoggedIn ? user.hasRole('Order fulfilment') : false;
```

Comment:
I think we can improve the condition a bit.
User can't exist without logging in. So if User object is there that means he/she is logged in or is isLoggedIn is true that means User exists.

Reply:
Also `isOrderFulfilmentUser` computed property should only check is user has `Order fulfilment` role or not. It shouldn't check if User is logged in or not.